src/client/src/orangehrmCoreOAuthPlugin/pages/registerOAuthClient/EditOAuthClient.vue

```diff
@@ -32,25 +32,40 @@
           <oxd-grid :cols="2" class="orangehrm-full-width-grid">
             <oxd-grid-item>
               <oxd-input-field
-                v-model="oAuthClient.clientId"
-                label="ID"
-                :rules="rules.clientId"
+                v-model="oAuthClient.name"
+                :label="$t('general.name')"
+                :rules="rules.name"
                 required
               />
             </oxd-grid-item>
             <oxd-grid-item>
               <oxd-input-field
-                v-model="oAuthClient.clientSecret"
-                label="Secret"
-                :rules="rules.clientSecret"
+                v-model="oAuthClient.redirectUri"
+                label="Redirect URI"
+                :rules="rules.redirectUri"
                 required
               />
             </oxd-grid-item>
             <oxd-grid-item>
+              <div class="orangehrm-module-field-row">
+                <oxd-text tag="p" class="orangehrm-module-field-label">
+                  Enable Client
+                </oxd-text>
+                <oxd-switch-input v-model="oAuthClient.enabled" />
+              </div>
+            </oxd-grid-item>
+            <oxd-grid-item class="--offset-row-3">
               <oxd-input-field
-                v-model="oAuthClient.redirectUri"
-                label="Redirect URI"
-                :rules="rules.redirectUri"
+                v-model="oAuthClient.clientId"
+                label="Client ID"
+                disabled
+              />
+            </oxd-grid-item>
+            <oxd-grid-item class="--offset-row-3">
+              <oxd-input-field
+                v-model="oAuthClient.clientSecret"
+                label="Client Secret"
```

Comment:
need to handle i18n
For this field we may have to show as `type="password"` and we should allow them to show/hide option

Reply:
i18n handled. leaving unresolved since password field not done yet